packages/components/psammead-timestamp/src/index.stories.jsx

```diff
@@ -1,26 +1,57 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
-import { text, withKnobs } from '@storybook/addon-knobs';
+import { text, select, withKnobs } from '@storybook/addon-knobs';
+import { latin } from '@bbc/gel-foundations/scripts';
+import * as typography from '@bbc/gel-foundations/typography';
 import notes from '../README.md';
 import Timestamp from '.';
 
+const styles = [
```

Comment:
Rather than having these hardcoded here? Could you maybe use the `typography` object to get the key values? Means if we were to ever add a new one, it would just work with a version bump